addon/-private/state-machine.js

```diff
@@ -80,9 +80,19 @@ export default EmberObject.extend({
     return transitions[currentStep];
   },
 
-  next() {
-    const next = this.peek();
-    return this.activate(next);
+  previous() {
```

Comment:
If the idea here is that `previous()` returns the name of the previous step without advancing, then I'd prefer we keep it similar to the `peek` naming -- maybe `peekPrevious` or something like that. `next()` advances to the next step, so I would expect `previous()` to do the same.

Reply:
I got your point. To avoid any ambiguity around, I purpose to rename `peek` in `peekNext`. That way we'll have consistency with `peekPrevious`.